WordPress/Classes/ViewRelated/Menus/Views/MenuItemSourceHeaderView.m

```diff
@@ -0,0 +1,113 @@
+#import "MenuItemSourceHeaderView.h"
+#import "WPStyleGuide.h"
+#import "WPFontManager.h"
+#import "MenuItem+ViewDesign.h"
+
+@import Gridicons;
+
+@interface MenuItemSourceHeaderView ()
+
+@property (nonatomic, strong) UIStackView *stackView;
```

Comment:
Since these are not meant to be rewritten, why not make them `readonly`?  Same for the view below this one.